.github/workflows/tag_and_release_protos.yml

```diff
@@ -57,7 +57,14 @@ jobs:
       image: ghcr.io/viamrobotics/canon:amd64
     strategy:
       matrix:
-        repo: ['viamrobotics/rdk','viamrobotics/app','viamrobotics/python-sdk','viamrobotics/rust-sdk','viamrobotics/rust-utils','viamrobotics/viam-cpp-sdk']
+        repo:
+          - viamrobotics/rdk
+          - viamrobotics/app
+          - viamrobotics/python-sdk
+          - viamrobotics/rust-sdk
```

Comment:
do we need to add the `viam-` prefix here? i guess it's working via redirect to the old name...